### ISSUE TITLE
Peer Review 1 Is Happening Now

### ISSUE THREAD

**Issue by aammd** [BIOL548O/Discussion#11]
hello @BIOL548O/all ,

**TL;DR : reply to this issue if you have a problem with the Peer Review process in this course.**

I've opened the first peer review issues. I'm sending this message to everyone in case a student has been left out.

If you think you are a student in the course, Please look to see if you have been notified that you are doing a Peer Review!  If you did **_not**_ receive any notification, please let me know ASAP! 

Alternatively if any of you who are joining us informally (ie you're not a student) **DID** get such a notification, that's also a problem! please let me know.

Notifications would have been in the form of an email, saying something like "You have been added to BIOL548O/Lastname_Firstname" or "you have been assigned to (something about issues) in BIOL548O/Lastname_Firstname". (those are approximate titles; if your message was not quite that don't worry).

**Comment by estum002** [comment]
Hi Andrew, 
Sorry I'm a bit confused - I received a confirmation for the person who is going to review my README, but do not know who's README I will be reviewing.
Thanks
Best,
Emilie

**Comment by robinycfang** [comment]
Hey @aammd ,

Just a quick question about the peer review sheet. The marking scale from 1 to 5...my understanding is that 5 is the best and 1 is the worst, or reverse?

Thanks!
Yuanchang

**Comment by aammd** [comment]
good question!  
1  = not good  
5 = good  

I'll be more explicit next time!

**Comment by AlexdeBruyn** [comment]
Hi! I've submitted the form for my peer review, but I can't figure out how to reply with a question about the project. How can I do that?

**Comment by aammd** [comment]
@AlexdeBruyn you can do that by replying to the issue on @mairindeith 's repository. Do you have access to that?

**Comment by aammd** [comment]
@BIOL548O/all Quick reminder to everyone to finish these today **before class** :smile_cat:
